NAMESPACE

```diff
@@ -3,7 +3,10 @@
 export(build.panel)
 export(build.psid)
 export(testPSID)
+export(get.variable.names)
 import(RCurl)
 import(SAScii)
 import(data.table)
 import(foreign)
+import(XML)
```

Comment:
im not a big fan of `XML` as it is quite a heavy dependency. if there is no alternative to it, then fine. maybe you can import just the one function that is needed?

Reply:
OK! I actually use a bunch of functions from XML, but I could fix it this way. I do not import XML and use XML::f() when I need function f. The only drawback is that this only works if the user has not previously downloaded XML then he will not be able to use the function get.variable.names. I introduced the line
`tryCatch(installed.packages()["XML",], error=function(e) print("ATTENTION: the package XML must be downloaded manually in order to use the function get.variable.names\n"))`
so that the user is warned and can download XML if he wishes to use this specific function.